Web/HTML1/MetaGurukul.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="ISO-8859-1">
+<title>Insert title here</title>
+</head>
+<body>
+	<form>
+	<fieldset form="">
+		<table>
+		<tr><td align="right"><button type="button">Help</td></tr>
+			<tr>
+				<td colspan=2>Metagurukul Account Registration</td>
+			</tr>
+			<tr>
+				<td align="right">Name:</td>
+				<td><input type="text" placeholder="First Name"><input
+					type="text" placeholder="Last Name"></td>
+			</tr>
+
+			<tr>
+				<td align="right">Gender:</td>
+				<td><select><option>male</option><option>female</option><option>others</option></select></td>
+			</tr>
+
+			<tr>
+				<td align="right">BirthDay:</td>
+				<td><select><option>January</option>
+				<option>February</option>
+				<option>March</option>
+				<option>April</option>
+				<option>May</option>
+				<option>June</option>
+				<option>July</option>
+				<option>August</option>
+				<option>September</option>
+				<option>October</option>
+				<option>November</option>
+				<option>December</option>
+				</select><input
+					type="text" placeholder="Day"><input type="text"
+					placeholder="Year"></td>
+			</tr>
+
+			<tr>
+				<td align="right">Country:</td>
+				<td><select><option>India</option></select></td>
+			</tr>
+
+			<tr>
+				<td colspan=2><hr>Select an ID and Password</td>
+			</tr>
+			<tr>
+				<td align="right">Metagurukul ID:</td>
+				<td><input type="text">@<select><option>metagurukul.com</option></select>
+					<button type="button">Check</button></td>
+			</tr>
+			<tr>
+				<td align="right">Password</td>
+				<td><input type="text"><font size="1" color="gray">password strength</font></td>
+			</tr>
+			<tr>
+				<td align="right">ReType Password</td>
+				<td><input type="text"></td>
+			</tr>
+			<tr><td colspan="2"><hr></td></tr>
+			<tr>
+				<td align="right">In case you forgot your ID or password...</td>
```

Comment:
ACTION: According to the screenshot this text should come above line but it is coming below line